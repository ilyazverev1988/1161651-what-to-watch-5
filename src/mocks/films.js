export default [
  {
    preview: `img/aviator.jpg`,
    nameFilm: `Aviator`,
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    filmCover: `img/bg-the-grand-budapest-hotel.jpg`,
    genre: `Thrillers`,
    releaseYear: 2014,
    descriptionFilm: `Friendship, metamorphosis, and adventure.`,
    commonScore: 8.9,
    numberOfVotes: 3,
    cast: `Everything we do is connected with zen: sex, bliss, moonlight, sorrow.`,
    producer: `Heu, orexis!`,
    duration: `1h 39m`,
    linkPreviewVideo: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.web`,
    linkFullVideo: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4 `
  },
  {
    preview: `img/dardjeeling-limited.jpg`,
    nameFilm: `Dardjeeling limited`,
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    filmCover: `img/bg-the-grand-budapest-hotel.jpg`,
    genre: `Sci-Fi`,
    releaseYear: 2016,
    descriptionFilm: `Spaces malfunction with adventure!`,
    commonScore: 7.9,
    numberOfVotes: 2,
    cast: `Never ransack a woodchuck.`,
    producer: `When the creator of zen acquires the stigmas of the aspect, the extend will know seeker.`,
    duration: `1h 40m`,
    linkPreviewVideo: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.web`,
    linkFullVideo: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4 `
  },
  {
    preview: `img/bohemian-rhapsody.jpg`,
    nameFilm: `Bohemian rhapsody`,
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    filmCover: `img/bg-the-grand-budapest-hotel.jpg`,
    genre: `Sci-Fi`,
    releaseYear: 2015,
    descriptionFilm: `Spaces malfunction with adventure!`,
    commonScore: 6.9,
    numberOfVotes: 2,
    cast: `Never ransack a woodchuck.`,
    producer: `When the creator of zen acquires the stigmas of the aspect, the extend will know seeker.`,
    duration: `1h 40m`,
    linkPreviewVideo: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.web`,
    linkFullVideo: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4 `
  },
  {
    preview: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    nameFilm: `Fantastic beasts the crimes of grindelwald`,
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    filmCover: `img/bg-the-grand-budapest-hotel.jpg`,
    genre: `Sci-Fi`,
    releaseYear: 2018,
    descriptionFilm: `Spaces malfunction with adventure!`,
    commonScore: 8.9,
    numberOfVotes: 3,
    cast: `Never ransack a woodchuck.`,
    producer: `When the creator of zen acquires the stigmas of the aspect, the extend will know seeker.`,
    duration: `1h 50m`,
    linkPreviewVideo: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.web`,
    linkFullVideo: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4 `
  },
  {
    preview: `img/johnny-english.jpg`,
    nameFilm: `Johnny english`,
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    filmCover: `img/bg-the-grand-budapest-hotel.jpg`,
    genre: `Sci-Fi`,
    releaseYear: 2019,
    descriptionFilm: `Spaces malfunction with adventure!`,
    commonScore: 9.9,
    numberOfVotes: 2,
    cast: `Never ransack a woodchuck.`,
    producer: `When the creator of zen acquires the stigmas of the aspect, the extend will know seeker.`,
    duration: `1h 30m`,
    linkPreviewVideo: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.web`,
    linkFullVideo: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4 `
  },
  {
    preview: `img/macbeth.jpg`,
    nameFilm: `Macbeth`,
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    filmCover: `img/bg-the-grand-budapest-hotel.jpg`,
    genre: `Sci-Fi`,
    releaseYear: 2020,
    descriptionFilm: `Spaces malfunction with adventure!`,
    commonScore: 5.9,
    numberOfVotes: 1,
    cast: `Never ransack a woodchuck.`,
    producer: `When the creator of zen acquires the stigmas of the aspect, the extend will know seeker.`,
    duration: `1h 33m`,
    linkPreviewVideo: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.web`,
    linkFullVideo: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4 `
  },
  {
    preview: `img/no-country-for-old-men.jpg`,
    nameFilm: `No country for old men`,
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    filmCover: `img/bg-the-grand-budapest-hotel.jpg`,
    genre: `Sci-Fi`,
    releaseYear: 2017,
    descriptionFilm: `Spaces malfunction with adventure!`,
    commonScore: 7.9,
    numberOfVotes: 2,
    cast: `Never ransack a woodchuck.`,
    producer: `When the creator of zen acquires the stigmas of the aspect, the extend will know seeker.`,
    duration: `1h 44m`,
    linkPreviewVideo: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.web`,
    linkFullVideo: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4 `
  },
  {
    preview: `img/war-of-the-worlds.jpg`,
    nameFilm: `War of the worlds`,
    poster: `img/the-grand-budapest-hotel-poster.jpg`,
    filmCover: `img/bg-the-grand-budapest-hotel.jpg`,
    genre: `Sci-Fi`,
    releaseYear: 2015,
    descriptionFilm: `Spaces malfunction with adventure!`,
    commonScore: 6.9,
    numberOfVotes: 1,
    cast: `Never ransack a woodchuck.`,
    producer: `When the creator of zen acquires the stigmas of the aspect, the extend will know seeker.`,
    duration: `1h 54m`,
    linkPreviewVideo: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.web`,
    linkFullVideo: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4 `
  },
];
