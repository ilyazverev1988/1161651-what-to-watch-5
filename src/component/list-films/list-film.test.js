import React from "react";
import renderer from "react-test-renderer";
import ListFilm from "./list-film";
import films from "../../mocks/films";
import {MemoryRouter} from "react-router-dom";

const noop = () => {};

describe(`Should ListFilm render correctly`, () => {
  it(`With filmActive`, () => {
    const tree = renderer
      .create(
          <MemoryRouter>
            <ListFilm
              filmActive={``}
              films={films}
              handleMouseEnterFilm={noop}
              handleMouseOverFilm={noop}
            />
          </MemoryRouter>, {
            createNodeMock: () => {
              return {};
            }
          }
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`No filmActive`, () => {
    const tree = renderer
      .create(
          <MemoryRouter>
            <ListFilm
              filmActive={1}
              films={films}
              handleMouseEnterFilm={noop}
              handleMouseOverFilm={noop}
            />
          </MemoryRouter>, {
            createNodeMock: () => {
              return {};
            }
          }
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

