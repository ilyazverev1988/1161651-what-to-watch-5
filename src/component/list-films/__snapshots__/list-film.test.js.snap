// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ListFilm render correctly No filmActive 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/1"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/1"
        onClick={[Function]}
      >
        Aviator1
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/2"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/2"
        onClick={[Function]}
      >
        Aviator2
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/3"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/3"
        onClick={[Function]}
      >
        Aviator3
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/4"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/4"
        onClick={[Function]}
      >
        Aviator4
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/5"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/5"
        onClick={[Function]}
      >
        Aviator5
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/6"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/6"
        onClick={[Function]}
      >
        Aviator6
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/7"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/7"
        onClick={[Function]}
      >
        Aviator7
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/8"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/8"
        onClick={[Function]}
      >
        Aviator8
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/9"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/9"
        onClick={[Function]}
      >
        Aviator9
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/10"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/10"
        onClick={[Function]}
      >
        Aviator10
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/11"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/11"
        onClick={[Function]}
      >
        Aviator11
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/12"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/12"
        onClick={[Function]}
      >
        Aviator12
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/13"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/13"
        onClick={[Function]}
      >
        Aviator13
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/14"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/14"
        onClick={[Function]}
      >
        Aviator14
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/15"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/15"
        onClick={[Function]}
      >
        Aviator15
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/16"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/16"
        onClick={[Function]}
      >
        Aviator16
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/17"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/17"
        onClick={[Function]}
      >
        Aviator17
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/18"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/18"
        onClick={[Function]}
      >
        Aviator18
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/19"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/19"
        onClick={[Function]}
      >
        Aviator19
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/20"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/20"
        onClick={[Function]}
      >
        Aviator20
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/21"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/21"
        onClick={[Function]}
      >
        Aviator21
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/22"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/22"
        onClick={[Function]}
      >
        Aviator22
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/23"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/23"
        onClick={[Function]}
      >
        Aviator23
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/24"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/24"
        onClick={[Function]}
      >
        Aviator24
      </a>
    </h3>
  </article>
</div>
`;

exports[`Should ListFilm render correctly With filmActive 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/1"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/1"
        onClick={[Function]}
      >
        Aviator1
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/2"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/2"
        onClick={[Function]}
      >
        Aviator2
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/3"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/3"
        onClick={[Function]}
      >
        Aviator3
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/4"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/4"
        onClick={[Function]}
      >
        Aviator4
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/5"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/5"
        onClick={[Function]}
      >
        Aviator5
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/6"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/6"
        onClick={[Function]}
      >
        Aviator6
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/7"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/7"
        onClick={[Function]}
      >
        Aviator7
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/8"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/8"
        onClick={[Function]}
      >
        Aviator8
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/9"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/9"
        onClick={[Function]}
      >
        Aviator9
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/10"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/10"
        onClick={[Function]}
      >
        Aviator10
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/11"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/11"
        onClick={[Function]}
      >
        Aviator11
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/12"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/12"
        onClick={[Function]}
      >
        Aviator12
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/13"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/13"
        onClick={[Function]}
      >
        Aviator13
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/14"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/14"
        onClick={[Function]}
      >
        Aviator14
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/15"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/15"
        onClick={[Function]}
      >
        Aviator15
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/16"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/16"
        onClick={[Function]}
      >
        Aviator16
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/17"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/17"
        onClick={[Function]}
      >
        Aviator17
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/18"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/18"
        onClick={[Function]}
      >
        Aviator18
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/19"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/19"
        onClick={[Function]}
      >
        Aviator19
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/20"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/20"
        onClick={[Function]}
      >
        Aviator20
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/21"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/21"
        onClick={[Function]}
      >
        Aviator21
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/22"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/22"
        onClick={[Function]}
      >
        Aviator22
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/23"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/23"
        onClick={[Function]}
      >
        Aviator23
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      href="/films/24"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseOut={[Function]}
        onMouseOver={[Function]}
      >
        <video
          height="175"
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/24"
        onClick={[Function]}
      >
        Aviator24
      </a>
    </h3>
  </article>
</div>
`;
